memorix-recordtypes/Bestanddeel.ttl

```diff
@@ -105,81 +106,81 @@ rt:Bestanddeel a memorix:Recordtype ,
                                       sh:maxCount 1 ;
                                       sh:order    4.0 ;
-                                      sh:path     bestanddeel:storageNumber ] ;
+                                      sh:path     file:storageNumber ] ;
     sh:property                     [ rdfs:label  "Interne opmerking"@nl ;
                                       dash:editor dash:TextAreaEditor ;
                                       sh:datatype xsd:string ;
-                                      sh:group    bestanddeel:managementGroup ;
+                                      sh:group    file:managementGroup ;
                                       sh:maxCount 1 ;
                                       sh:order    5.0 ;
-                                      sh:path     bestanddeel:remark ] ;
+                                      sh:path     file:internalRemarks ] ;
     sh:property                     [ rdfs:label   "Duplicaat (kopie op film of microfiche)"@nl ;
                                       rdfs:comment "Kopieën van dit bestanddeel op film of microfiche."@nl ;
                                       sh:message   "Selecteer een duplicaat (filmnummer of microfiche)"@nl ;
                                       dash:editor  memorix:LinkedRecordEditor ;
-                                      sh:group     bestanddeel:managementGroup ;
+                                      sh:group     file:managementGroup ;
                                       sh:maxCount  1 ;
                                       sh:nodeKind  sh:IRI ;
                                       sh:or        ( [ sh:class rt:SaaDuplicaat ] ) ;
                                       sh:order     6.0 ;
-                                      sh:path      bestanddeel:duplicate ] ;
+                                      sh:path      file:duplicate ] ;
     sh:property                     [ rdfs:label   "Zie ook - verwijzing naar ander bestanddeel"@nl ;
                                       rdfs:comment "Sommige bestanddelen verwijzen naar andere bestanddelen."@nl ;
                                       sh:message   "Selecteer een ander bestanddeel"@nl ;
                                       dash:editor  memorix:LinkedRecordEditor ;
-                                      sh:group     bestanddeel:managementGroup ;
+                                      sh:group     file:managementGroup ;
                                       sh:maxCount  1 ;
                                       sh:nodeKind  sh:IRI ;
-                                      sh:or        ( [ sh:class rt:SaaBestanddeel ] ) ;
+                                      sh:or        ( [ sh:class rt:File ] ) ;
                                       sh:order     7.0 ;
-                                      sh:path      rdfs:seeAlso ] ; 
+                                      sh:path      rdfs:seeAlso ] ;  # todo Ivo, is hier niet iets voor in rico?
```

Comment:
Nee, want er deze hele specifieke die het ook al doet. Daarom hebben ze hem niet in Rico gedefinieerd.